traveler-swift-core/TravelerKit/Models/OrderResult.swift

```diff
@@ -64,12 +67,34 @@ public struct OrderResult: Decodable {
         }
 
         var orders = self.orders
+        var sequence = self.ordersSequence
 
         for (index, order) in result.orders {
             orders[index] = order
+            sequence[order] = index
         }
 
-        return OrderResult(total: result.total, orders: orders, fromDate: result.fromDate, toDate: result.toDate)
+        return OrderResult(total: result.total, orders: orders, ordersSequence: sequence, fromDate: result.fromDate, toDate: result.toDate)
+    }
+
+    /**
+     Updates `Order` in orders in self.
+
+     - Parameters:
+     - order: The `Order` to be updated
+
+     - Returns: The index in which the `Order` was updated or a `OrderResultError` if `Order` is not in `OrderResult`
```

Comment:
it doesnt return an error

Reply:
You have to document throwing separately 

- Throws: An `OrderResultError` if the `Order` was not part of the result